spec/double_entry/reporting_spec.rb

```diff
@@ -201,6 +201,34 @@
         end
       end
 
+      context 'filtering by key with multiple values' do
+        let(:code) { :spend }
+        let(:filter) do
+          [
+            :metadata => {
+              :category => category,
```

Comment:
I wonder if it makes sense to allow a list of categories in the filter? But we would have to make the decision if an entry had to match *all categories* or just *any category* in that list. Perhaps this is out of scope at the moment...